src/makeInfinityAdder.js

```diff
@@ -5,6 +5,21 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let acc = 0;
+
+  return function adder(param) {
+    if (param !== undefined) {
```

Comment:
The condition `param !== undefined` is correct for checking if a parameter is provided. However, ensure that the function is intended to handle only numeric values. If non-numeric values are passed, it might lead to unexpected behavior. Consider adding a type check to ensure `param` is a number.